### PR TITLE
docs: added a basic vweb getting started info

### PR DESCRIPTION
**Pull request by Zerotask**
To allow an easy start with vweb, I summarized the basics to get a vweb web server running and how to define endpoints. At a later point it would be better to have an extensive documentation inside the `doc` folder.

## Preview
image
image

### CODE REVIEW

File: vlib/vweb/README.md [hunk 1]
```diff
@@ -61,3 +61,77 @@ That means that the template automatically has access to that action's entire en
 ### Deploying vweb apps
 
 Everything, including HTML templates, is in one binary file. That's all you need to deploy.
+
+## Getting Started
+
+To start with vweb, you have to import the module `vweb` first.
```
Comment (JalonSolov) [inline vlib/vweb/README.md:67]: First, import the `vweb` module.

File: vlib/vweb/README.md [hunk 2]
```diff
@@ -61,3 +61,77 @@ That means that the template automatically has access to that action's entire en
...
+After it, it's recommended to define a struct to access `vweb.Context`.
```
Comment (JalonSolov) [inline vlib/vweb/README.md:68]: After the import, define a struct to hold the `vweb.Context` and any other variables your program will need.  For this example, we will only keep the `vweb.Context` variable.

Comment (Zerotask) [reply vlib/vweb/README.md:68]: Thank you for your updates. I would suggest to keep it as it is since it's just meant as a little info. There is a lot of room for improvements.
For the future it's better to make a fine structured separate documentation inside the `doc` folder.

File: vlib/vweb/README.md [hunk 3]
```diff
@@ -61,3 +61,77 @@ That means that the template automatically has access to that action's entire en
...
+
+**Example:**
+```v ignore
+[post]
+fn (mut app App) world() vweb.Result {
+	return app.text('World')
+}
+```
+
+Instead of using the auto-mapping, you can
```
Comment (JalonSolov) [inline vlib/vweb/README.md:109]: Instead of using auto-mapping, you can

File: vlib/vweb/README.md [hunk 4]
```diff
@@ -61,3 +61,77 @@ That means that the template automatically has access to that action's entire en
...
+**Example:**
+```v ignore
+[post]
+["/foo"]
+fn (mut app App) world() vweb.Result {
+	return app.text('World')
+}
+```
+
+To pass a param to an endpoint, you simply define it inside an attribute, e. g. `['/hello/:user]`.
```
Comment (JalonSolov) [inline vlib/vweb/README.md:122]: `parameter` instead of just `param`

File: vlib/vweb/README.md [hunk 5]
```diff
@@ -61,3 +61,77 @@ That means that the template automatically has access to that action's entire en
...
+After it, you have to add it as a function parameter.
```
Comment (JalonSolov) [inline vlib/vweb/README.md:123]: After it is defined in the attribute, you have to add it as a function parameter.

File: vlib/vweb/README.md [hunk 6]
```diff
@@ -61,3 +61,77 @@ That means that the template automatically has access to that action's entire en
...
+
+**Example:**
+```v ignore
+['/hello/:user']
+fn (mut app App) hello_user(user string) vweb.Result {
+	return app.text('Hello $user')
+}
+```
+
+You can read several request data like the request headers
```
Comment (JalonSolov) [inline vlib/vweb/README.md:133]: You have access to the raw request data such as headers

File: vlib/vweb/README.md (hunk 2 above)
Comment (JalonSolov) [reply vlib/vweb/README.md:68]: Even if you don't mention other variables, I would still change the first line.  Perhaps to this

After the import, define a struct to hold `vweb.Context`.

File: vlib/vweb/README.md (hunk 6 above)
Comment (Zerotask) [reply vlib/vweb/README.md:133]: Many of your suggestions are subjectively perceived.

File: vlib/vweb/README.md (hunk 3 above)
Comment (spytheman) [reply vlib/vweb/README.md:109]: it is not mapping, it is routing

Comment (spytheman) [reply vlib/vweb/README.md:109]: mapping in this context will be confusing imho

File: vlib/vweb/README.md (hunk 6 above)
Comment (spytheman) [reply vlib/vweb/README.md:133]: No they are not. They are objectively better than your text, and you are being obstinate @Zerotask .

Comment (spytheman) [reply vlib/vweb/README.md:133]: `You can read several request data like the request headers` this was hard to parse and understand for me too.

File: vlib/vweb/README.md (hunk 3 above)
Comment (Zerotask) [reply vlib/vweb/README.md:109]: Function names are mapped to the routing. If you use the attribute to define a specific path, then you could talk about routing.
I could describe this point better.

File: vlib/vweb/README.md (hunk 6 above)
Comment (Zerotask) [reply vlib/vweb/README.md:133]: We are talking about opinions. You can't say it's objectively better neither can you say I'm obstinate.

Comment (spytheman) [reply vlib/vweb/README.md:133]: I already did, and it is objectively better - shorter and easier to understand.

Comment (spytheman) [reply vlib/vweb/README.md:133]: image

Comment (spytheman) [reply vlib/vweb/README.md:133]: I can make a compromise - they are `objectively` and `subjectively` better.
